src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DrawLineAdapter.java

```diff
@@ -8,11 +8,14 @@
 /**
  * driver adapter to drawer with several bugs.
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+public class DrawLineAdapter extends DrawPanelController implements Job2dDriver {
```

Comment:
Inheritance is not used (no super drawLine calls). Should be removed!
The name is better but it lacks a driver aspect.